_sass/_blocks.scss

```diff
@@ -1262,6 +1262,41 @@
   }
 }
 
+
+
+
+.sectionWrapper {
+  padding-bottom: spacing(2);
+
+
+    display: flex;
+    justify-content: space-between;
+    flex-direction: column;
+
+    @include mq($from: desktop) {
+      flex-direction: row;
+    }
+
+    h3 {
+      font-size: map-get($font-scale, 4);
+      line-height: 1;
+      font-weight: 400;
+
+      @include mq($from: desktop) {
+        font-size: map-get($font-scale, 5);
+      }
+
+      @include mq($from: wide) {
+        font-size: map-get($font-scale, 10);
+      }
+    }
+
+}
+
+
+
+
+
```

Comment:
I've noticed you like to add a lot of newlines to separate some sections of the scss, maybe define a standard and use it throughout? I think 2 maybe 3 lines separation between any text should definitely separate things visually enough while still making things concise no?

Reply:
Yes i agree. i usually add empty line so i know were my focus code is starting and ending. I am to remember to delete that after testing